package_drupal_script.sh

```diff
@@ -17,16 +17,13 @@ export GZIP=-9
 
```

Comment:
The changes here make sense:

- Improve bash variable use with quoted strings.
- Using pushd/popd.
- Removing unnecessary conditionals.